README.md

```diff
@@ -1,4 +1,48 @@
-# Pubkey Encrypt
-Drupal 8 Module for adding Credentials-based Public Key Encryption support into the Field Encrypt module. The encryption mechanism is based on ownCloud's Data Encryption Model.
+# Pubkey Encrypt for Drupal 8
 
 [![Build Status](https://travis-ci.org/d8-contrib-modules/pubkey_encrypt.svg?branch=8.x)](https://travis-ci.org/d8-contrib-modules/pubkey_encrypt)
+
+Provides support for encrypting data with users login-credentials. The mechanism is based on ownCloud's Data Encryption Model.
+
+Pubkey Encrypt leverages the Key & Encrypt modules for integrating it's encryption mechanism into a Drupal website. Accordingly, **the module generates an Encryption Profile for each role in the website.** This allows an administrator to select the set of users whose login-credentials should be used during the encryption/decryption processes. For example, if an administrator choses the "Premium User Encryption Profile" while encrypting the data, then the login-credentials of all users from the "Premium User" role would get used. Consequently, only these users would then be able to decrypt the encrypted data.
+
+* For encrypting field data using this mechanism, Field Encrypt module should be used along with this one.
+* For encrypting files using this mechanism, Encrypted Files module should be used along with this one.
```

Comment:
We renamed this to File Encrypt --- https://github.com/d8-contrib-modules/file_encrypt